src/controllers/dynakube/istio/controller.go

```diff
@@ -66,7 +52,7 @@ func (c *Controller) initializeIstioClient(config *rest.Config) (istioclientset.
 // creating/deleting VS & SE for external communications
 func (c *Controller) ReconcileIstio(instance *dynatracev1beta1.DynaKube) (updated bool, err error) {
```

Comment:
nitpick: the `istio` Controller is not really a controller, so in this https://github.com/Dynatrace/dynatrace-operator/pull/462 it will be renamed to Reconciler